lib/python/vdm/server/HTTPListener.py

```diff
@@ -458,6 +460,41 @@ def map_deployment_users(request, user):
     return deployment_user[0]
 
 
+def ignore_sighup():
+    signal.signal(signal.SIGHUP, signal.SIG_IGN)
+    signal.signal(signal.SIGINT, signal.SIG_IGN)
+
+def start_local_server(deploymentcontents):
+    filename = os.path.join(PATH, 'deployment.xml')
+    deploymentfile = open(filename, 'w');
+    deploymentfile.write(deploymentcontents)
+    deploymentfile.close()
+    voltdb_dir = os.path.realpath(os.path.join(MODULE_PATH, '../../../..', 'bin'))
+    voltdb_cmd = [ 'nohup', os.path.join(voltdb_dir, 'voltdb'), 'create', '-d', filename ]
+    outfilename = os.path.join(PATH, 'voltserver.output')
+    outfile = open(outfilename, 'w')
+
+    # Start server in a separate process
+    voltserver = subprocess.Popen(voltdb_cmd, stdout=outfile, stderr=subprocess.STDOUT,
+                                  preexec_fn=ignore_sighup)
+
+    grep_cmd = [ 'grep', 'Server completed initialization', outfilename ]
+    grep_retcode = subprocess.call(grep_cmd)
```

Comment:
you use tell() and seek() so that you can start searching from where you last left off